### PR TITLE
Correct color sorting

### PR DESCRIPTION
**Pull request by mlms13**

### CODE REVIEW

**Comment by fponticelli** [comment]
I think it is awesome! ... but no :)

**Comment by longcao** [comment]
:+1:

**Comment by mrvisser** [comment]
Is this sort stable? What if it's a particularly warm day?

**Comment by fponticelli** [comment]
I think it has all the features

**Comment by mlms13** [comment]
No, these are edge cases I hadn't considered. I understand the rejection now.

File: test/thx/color/TestColor.hx [hunk 1]
```diff
@@ -20,6 +20,10 @@ class TestColor {
     assertEquals(Color.red.withAlpha(0), Color.parse("0x00ff0000"));
   }
 
+  public function testSort() {
+    Assert.same("rainbow", Color.sort());
```
Comment (fponticelli) [inline test/thx/color/TestColor.hx]: `Assert.equals()` is better in this case, slightly faster.

File: src/thx/color/Color.hx [hunk 2]
```diff
@@ -439,4 +439,8 @@ class Color {
     names.set("yellowgreen", yellowgreen = 0x9acd32);
     names.set("yellow green", yellowgreen);
   }
-}
\ No newline at end of file
+
+  public static function sort() {
+    return "rainbow";
```
Comment (fponticelli) [inline src/thx/color/Color.hx]: maybe `rainbow` should be capitalized?
